lib/stubs.ts

```diff
@@ -0,0 +1 @@
+declare module 'adbkit-apkreader';
```

Comment:
what is the purpose of this stub?

Reply:
this tells TS that "yes, this module exists, and no, it does not have any types, so treat it as `any`".

Reply:
maybe add a comment there to make it cleaner?